packages/cashier/src/app-theme-content.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Routes from 'Containers/routes';
+import { observer, useStore } from '@deriv/stores';
+import { useTheme } from '@deriv/ui';
+
+const AppThemeContent = () => {
```

Comment:
Why the name `AppThemeContent`? 🤔

Reply:
@nijil-deriv mmm because component contain theme implementation for cashier package. What naming would you like to suggest instead?

Reply:
@nijil-deriv will change it to AppContent